lib/Parser/parse.cpp

```diff
@@ -7708,7 +7708,7 @@ void PidRefStack::TrackAssignment(charcount_t ichMin, charcount_t ichLim)
         {
             return;
         }
-        Assert(this->GetIchLim() >= ichLim);
+        Assert(ichMin <= this->GetIchMin() && this->GetIchLim() <= ichLim)
```

Comment:
are you looking at lib\common\core\Assertions.h? So debug build is fine without semicolon, but non-dbg will have problem.